meta-mender-core/recipes-mender/mender-client/files/mender-client-resize-data-part.sh.in

```diff
@@ -4,8 +4,8 @@ set -e
 
 # All sizes are in blocks of 512 bytes
 
-MENDER_STORAGE_DEVICE=@MENDER_STORAGE_DEVICE@
 MENDER_DATA_PART=@MENDER_DATA_PART@
+MENDER_STORAGE_DEVICE=@MENDER_STORAGE_DEVICE@
```

Comment:
Why was this needed?

Reply:
The variable are interpreted top-down in shell.

And whe do:

```
MENDER_DATA_PART=$(blkid -L data)
MENDER_STORAGE_DEVICE="/dev/$(basename $(readlink -f "/sys/class/block/$(basename $MENDER_DATA_PART)/..))"
```

The readlink is a bit ugly we could do something like this instead:
```
MENDER_STORAGE_DEVICE="/dev/$(lsblk -no PKNAME $MENDER_DATA_PART)"
```
But this would required the lsblk utility

Reply:
Okey.